src/containers/pages/InterventionPlan/IRS/plan/index.tsx

```diff
@@ -1,79 +1,1419 @@
+      'New Plan';
+
+    const breadCrumbProps = this.getBreadCrumbProps(this.props, pageLabel);
+
+    let planTableProps: DrillDownProps<any> | null; // todo - type with DrillDownProps
+    if (isFinalizedPlan) {
+      planTableProps = this.getFinalizedPlanTableProps(this.props);
+    } else {
+      planTableProps = this.getDrilldownPlanTableProps(this.state);
+    }
+
+    const onSetPlanNameChange = (e: any) => {
+      this.onSetPlanNameChange(e);
+    };
+    const onSetPlanStartDateChange = (e: any) => {
+      this.onSetPlanStartDateChange(e);
+    };
+    const onSetPlanEndDateChange = (e: any) => {
+      this.onSetPlanEndDateChange(e);
+    };
+    const onSelectCountryChange = (e: any) => {
+      this.onSelectCountryChange(e);
+    };
+    const onStartPlanFormSubmit = (e: any) => {
+      this.onStartPlanFormSubmit(e);
+    };
+
+    if (isStartingPlan && newPlan) {
+      const { plan_effective_period_end, plan_effective_period_start, plan_title } = newPlan;
+      return (
+        <div className="mb-5">
+          <Helmet>
+            <title>IRS: New Plan</title>
+          </Helmet>
+          <HeaderBreadcrumbs {...breadCrumbProps} />
+          <Row>
+            <Col>
+              <h2 className="page-title">New IRS Plan</h2>
+            </Col>
+          </Row>
+          <Row>
+            <Col xs="6">
+              <Form>
+                <FormGroup>
+                  <Label for="set-plan-name">Plan Name</Label>
+                  <Input
+                    id="set-plan-name"
+                    onChange={onSetPlanNameChange}
+                    placeholder={plan_title}
+                  />
+                </FormGroup>
+                <FormGroup>
+                  <Label for="set-plan-start-date">Start Date</Label>
+                  <Input
+                    id="set-plan-start-date"
+                    onChange={onSetPlanStartDateChange}
+                    defaultValue={plan_effective_period_start}
+                    type="date"
+                  />
+                </FormGroup>
+                <FormGroup>
+                  <Label for="set-plan-end-date">End Date</Label>
+                  <Input
+                    id="set-plan-end-date"
+                    onChange={onSetPlanEndDateChange}
+                    defaultValue={plan_effective_period_end}
+                    type="date"
+                  />
+                </FormGroup>
+                <FormGroup>
+                  <Label for="select-plan-country">Select a Country</Label>
+                  <Input
+                    id="select-plan-country"
+                    defaultValue={planCountry}
+                    name="select-plan-country"
+                    onChange={onSelectCountryChange}
+                    type="select"
+                  >
+                    <option>...</option>
+                    {/* <option value="BW">Botswana</option> */}
+                    <option value="NA">Namibia</option>
```

Comment:
Should we load these options from the settings module, or somewhere that allows for easy configuration?

Reply:
Done at `21bbddb`